assets/src/blocks/PageHeader/PageHeaderBlock.js

```diff
@@ -0,0 +1,49 @@
+const { registerBlockVariation } = wp.blocks;
+const { __ } = wp.i18n;
+
+export const registerPageHeaderBlock = () => {
+
+  let classname = 'is-pattern-p4_page-header';
```

Comment:
Do you prefer to use `_` for any particular reason, or it's just a typo mistake?

Reply:
ah no sorry, didn't update after the change in the pattern report PR :facepalm: